static/install.txt

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+if test -z "$BASH_VERSION"; then
+  echo "Please run this script using bash, not sh or any other shell." >&2
+  exit 1
+fi
+
+install() {
+  set -euo pipefail
+
+  dst_dir="${K14SIO_INSTALL_BIN_DIR:-/usr/local/bin}"
+
+  if [ -x "$(command -v wget)" ]; then
+    dl_bin="wget -nv -O-"
+  else
+    dl_bin="curl -s -L"
+  fi
+
+  shasum -v 1>/dev/null 2>&1 || (echo "Missing shasum binary" && exit 1)
+
+  ytt_version=v0.30.0
+  kbld_version=v0.27.0
+  kapp_version=v0.34.0
+  kwt_version=v0.0.6
+  imgpkg_version=v0.2.0
+  vendir_version=v0.12.0
```

Comment:
Wondering if this could be replaced eventually by a script to query the latest released version for each tool? Not saying it should happen as part of this pull request as it's out of scope, but might help with having to manually update versions.

Reply:
Agreed, since we update this with every release a script would reduce manual steps.

Reply:
hard coding was done intentionally to avoid making a mess with parsing release notes for checksum and accessing github api (there is actually no way to determine "true" latest version without parsing semvers and sorting; github by default uses latest as latest to be published which does not work when publishing patches for earlier versions).

separately, i typically favor this approach because it "distributes" expected checksum to multiple places. if "somebody" screws around with uploaded binary somehow, and github release notes, checksum here is present in git history. but what if we still want to automate... well it just becomes a matter of making a script somewhere that commits resolved version and checksum here (as close to the time of publish as possible). same script should of course update homebrew as well.